<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deusto Messenger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="labelNombreUsuario">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>210</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelContrasea">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>300</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Contraseña</string>
      </property>
     </widget>
     <widget class="QPushButton" name="iniciarSesionBtn">
      <property name="geometry">
       <rect>
        <x>372</x>
        <y>400</y>
        <width>251</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>INICIAR SESION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registrarNuevoUsuarioBtn">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>450</y>
        <width>251</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>REGISTRAR NUEVO USUARIO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_TituloRegistro_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>130</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>INICIO SESION</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_UsuarioInicioSes">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhEmailCharactersOnly</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_ContraInicioSes">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhHiddenText|Qt::InputMethodHint::ImhSensitiveData</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_ErrorInicioSes">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>510</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Los datos introducidos son incorrectos</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextInteractionFlag::NoTextInteraction</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Contactos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Chat</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>511</width>
        <height>541</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>165</number>
      </attribute>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QTextEdit" name="textEnviar">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>610</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Mensaje</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonEnviar">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>610</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enviar</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetContactos">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>381</width>
        <height>541</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonChatear">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>610</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Chatear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAadir">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>610</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEliminarContacto">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>610</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBtnAnadirUsu">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>610</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Añadir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="labelNombreADD">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNombreADD">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>281</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelEmailADD">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear grupo</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTelefonoADD">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Descripcion</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTelefonoADD">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>281</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAadirADD">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>231</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Añadir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="registro">
     <widget class="QTextEdit" name="textEdit_EmailRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhEmailCharactersOnly</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_TituloRegistro">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>REGISTRO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_EmailRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>E-Mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_TlfRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Telefono</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_TlfRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhDialableCharactersOnly</set>
      </property>
      <property name="placeholderText">
       <string>+34</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_NombreRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_FechaRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>430</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fecha Nacimiento</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ContraRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>530</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Contraseña</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_NombreRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhNone</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_ContraRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>560</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhHiddenText|Qt::InputMethodHint::ImhSensitiveData</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_FechaRegistro">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>460</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhDate</set>
      </property>
      <property name="placeholderText">
       <string>YYYY-MM-DD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ConfirmarRegistro">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>640</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>CONFIRMAR</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
